experimental/packages/opentelemetry-sdk-metrics-base/src/aggregator/Histogram.ts

```diff
@@ -37,19 +37,33 @@ function createNewEmptyCheckpoint(boundaries: number[]): Histogram {
     },
     sum: 0,
     count: 0,
+    hasMinMax: false,
+    min: Infinity,
+    max: -1
   };
 }
 
+function hasMinMax(histogram: Histogram): boolean{
```

Comment:
```suggestion
function hasMinMax(histogram: Histogram): boolean {
```

Reply:
Ah thanks, forgot to auto-format. :sweat: 
Also realized that I don't need this function anymore and I can easily inline it. It was a leftover from me trying things out. I reformatted the file and inlined that function in [031219f](https://github.com/open-telemetry/opentelemetry-js/pull/3032/commits/031219f43405aea3e5ccc8b8d66b02ef98d0c350). :slightly_smiling_face: